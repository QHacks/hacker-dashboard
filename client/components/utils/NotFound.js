import React from 'react';
import { Redirect } from 'react-router-dom';

export default ({ location }) => (
    <Redirect to={{
        pathname: '/',
        state: {
            from: location,
            message: 'Page not found'
        }
    }}/>
);
