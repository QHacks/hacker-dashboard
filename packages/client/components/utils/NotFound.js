import { Redirect } from "react-router-dom";
import React from "react";

export default ({ location }) => (
  <Redirect
    to={{
      pathname: "/",
      state: {
        from: location,
        message: "Page not found!"
      }
    }}
  />
);
