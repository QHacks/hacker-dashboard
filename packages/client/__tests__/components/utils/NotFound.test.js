import React from "react";
import NotFound from "../../../components/utils/NotFound";
import { shallow } from "enzyme";

describe("NotFound", () => {
  it("renders correctly", () => {
    expect(shallow(<NotFound />)).toMatchSnapshot();
  });
});
