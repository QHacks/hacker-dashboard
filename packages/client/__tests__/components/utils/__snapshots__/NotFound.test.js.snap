// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders correctly 1`] = `
<Redirect
  push={false}
  to={
    Object {
      "pathname": "/profile",
      "state": Object {
        "alert": Object {
          "message": "We couldn't find the page you were looking for.",
          "status": "Oops!",
          "type": "info",
        },
        "from": undefined,
      },
    }
  }
/>
`;
